### DISCUSSION TITLE
Tooltip anchor

### DISCUSSION THREAD

**Discussion by DarkCTO** [zauberzeug/nicegui#3018] [Ideas / Feature Requests]
We already have tooltip, however I cannot set the anchor to be on the center right for example.

I think it doesn't cost much to add this prop

```py
ui.item_section('Example').tooltip('An example')
```

**Comment by falkoschindler** [comment]
Hi @DarkCTO,

The `.tooltip` method is a bit limited. But you can also nest a `ui.tooltip` element, which supports all of [QTooltip](https://quasar.dev/vue-components/tooltip)'s props:

```py
with ui.label('Example'):
    ui.tooltip('An example').props('anchor="bottom right"')
```

**Comment by DarkCTO** [comment]
Got it. Thank you very much.
